mora/service/engagement.py

```diff
@@ -43,14 +43,15 @@ def list_details(type, id):
         "association": false,
         "engagement": true,
         "role": false,
-        "leave": true
+        "leave": true,
+        "manager": false
       }
 
-    The value above informs you that 'association', 'engagement', 'role' and
-    'leave' are valid for this entry, and that no entry exists at any time
-    for 'association' and 'role', whereas 'engagement' and 'leave' have at
-    least one entry either in the past, present or future.
-
+    The value above informs you that 'association', 'engagement', 'role',
+    'leave' and 'manager' are valid for this entry, and that no entry exists
+    at any time for 'association', 'role' and 'manager', whereas
+    'engagement' and 'leave' have at least one entry either in the past,
+    present or future.
```

Comment:
This is getting rather long-winded, I'd say 🙂 Perhaps we should rephrase it to just describe one or two `true` values, and describe everything else as “the others”…